benchmarks/bin/binary_decode_packed.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:math';
+import 'dart:typed_data';
+
+import 'package:fixnum/fixnum.dart';
+import 'package:protobuf_benchmarks/benchmark_base.dart';
+import 'package:protobuf_benchmarks/generated/packed_fields.pb.dart';
+
+PackedFields? sink;
```

Comment:
It seems this global `sink` is never used. That will make TFA remove it, which in turn makes the results of calls like `message.writeToBuffer()` unused, which may make optimizer remove optimize things away.

Could you add a `if (int.parse('1') == 0) print(sink)` to `main` to avoid this?

Reply:
Good catch -- I wanted to do this but then forgot..

With `print(sink)` VM benchmarks are the same as before, but in Wasm both before and after the benchmarks get much slower. The new version is still faster. I updated the PR description with new benchmarks.